pcawg_metadata_parser/parse_gnos_xml.py

```diff
@@ -799,6 +827,141 @@ def read_train2_bams(filename):
     return train2_bams
 
 
+def read_santa_cruz_entries(filename, santa_cruz_freeze_entry_tsv):
```

Comment:
generating santa_cruz_freeze_entry_tsv does not actually need to be in this script, as this is really just a one time thing because the freeze will never change. Let's factor this out later, leave it in for now for the interest of time